x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/scalar/conditional/CaseTests.java

```diff
@@ -203,35 +204,6 @@ public static Iterable<Object[]> parameters() {
         );
     }
 
-    @Override
-    protected void assertSimpleWithNulls(List<Object> data, Block value, int nullBlock) {
-        if (nullBlock == 0) {
-            if (data.size() == 2) {
-                assertThat(value.isNull(0), equalTo(true));
-            } else if (data.size() > 2) {
-                assertThat(toJavaObject(value, 0), equalTo(data.get(2)));
-            }
-            return;
-        }
-        if (((Boolean) data.get(0)).booleanValue()) {
-            if (nullBlock == 1) {
-                super.assertSimpleWithNulls(data, value, nullBlock);
-            } else {
-                assertThat(toJavaObject(value, 0), equalTo(data.get(1)));
-            }
-            return;
-        }
-        if (nullBlock == 2) {
-            super.assertSimpleWithNulls(data, value, nullBlock);
-        } else {
-            if (data.size() > 2) {
-                assertThat(toJavaObject(value, 0), equalTo(data.get(2)));
-            } else {
-                super.assertSimpleWithNulls(data, value, nullBlock);
-            }
-        }
-    }
-
```

Comment:
These are covered by the other tests, though those should be ported to test cases.